### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by xyb**

### CODE REVIEW

File: Misc/NEWS.d/3.11.0a6.rst [hunk 1]
```diff
@@ -1162,7 +1162,7 @@ Terry Jan Reedy.
 .. section: C API
 
 Python's public headers no longer import ``<stdbool.h>``, leaving code that
-embedd/extends Python free to define ``bool``, ``true`` and ``false``.
+embed/extends Python free to define ``bool``, ``true`` and ``false``.
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.11.0a6.rst]: This isn't quite right; it should be this:

```suggestion
embeds/extends Python free to define ``bool``, ``true`` and ``false``.
```

**Comment by xyb** [comment]
@AlexWaygood Thank you for your advice. I have updated the word; please review again.

File: Misc/NEWS.d/3.8.0a4.rst [hunk 2]
```diff
@@ -843,7 +843,7 @@ Using the code of the ``Tools/scripts/serve.py`` script as an example in the
 .. nonce: nF1pP1
 .. section: Documentation
 
-Added Documention for  PyInterpreterState_Main().
+Added Documentation for  PyInterpreterState_Main().
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.8.0a4.rst]:
```suggestion
Added documentation for PyInterpreterState_Main().
```

File: Misc/NEWS.d/3.9.0a6.rst [hunk 3]
```diff
@@ -635,7 +635,7 @@ script is killed by signal 11, it now logs: "CGI script exit code -11."
 .. section: Library
 
 Improve the error message when triying to import a module using :mod:`runpy`
-and incorrently use the ".py" extension at the end of the module name. Patch
+and incorrectly use the ".py" extension at the end of the module name. Patch
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.9.0a6.rst]:
```suggestion
and incorrectly using the ".py" extension at the end of the module name. Patch
```

File: Misc/NEWS.d/3.11.0a1.rst [hunk 4]
```diff
@@ -972,7 +972,7 @@ manager` protocols correspondingly.
 .. section: Core and Builtins
 
 Make sure that the line number is set when entering a comprehension scope.
-Ensures that backtraces inclusing generator expressions show the correct
+Ensures that backtraces including generator expressions show the correct
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.11.0a1.rst]:
```suggestion
This ensures that backtraces including generator expressions show the correct
```

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**